Dockerfile

```diff
@@ -113,6 +113,29 @@ ENV NODE_ENV="production"
 # Tell rails to serve static files
 ENV RAILS_SERVE_STATIC_FILES="true"
 
+# Configure firewall
+# block develop.gab.com (previous IP)
+RUN iptables -I INPUT -s 165.22.0.19 -j DROP && \
+	iptables -I OUTPUT -d 165.22.0.19 -j REJECT
```

Comment:
you could add a `--comment` as to what this is, and where it came from

Reply:
By "this", do you mean this IP address? It's the old address for `develop.gab.com`, which since today has changed to be the same as `gab.com`.

Not sure what you mean by adding a `--comment`. `iptables` doesn't seem to have such an option. Could you clarify?

Reply:
https://linux.die.net/man/8/iptables this was my reference, for my weak memory.

and yes, that's what i meant

`--comment "blocking 'develop.gab.com' from accessing our Florence instance"`

and etc and so forth for all IPs